src/Directory.Build.props

```diff
@@ -6,7 +6,7 @@
     <MSBuildAllProjects>$(MSBuildAllProjects);$(MSBuildThisFileDirectory)..\Directory.Build.props</MSBuildAllProjects>
   </PropertyGroup>
 
-  <Import Project="$(MSBuildThisFileDirectory)..\Directory.Build.props" />
+  <Import Project="..\Directory.Build.props" />
```

Comment:
Why is one import value absolute and the other is relative?

Reply:
@tmat had brought in this change.

Reply:
Which one is absolute? ```<Import>``` paths do not need to be absolute as they are evaluated relative to the containing file.

Reply:
I understand that functionally this does not matter; I'm commenting on this stylistically.
So this one is not absolute? Import Project="$(RepoToolsetDir)Settings.props"

Reply:
That one points to a directory that's not in the source repo (it's in the toolset nuget package)

Reply:
in other words in this case ```$(RepoToolsetDir)Settings.props``` is not the same as ```Settings.props``` while in the previous case ```$(MSBuildThisFileDirectory)..\Directory.Build.props``` is equivalent to ```..\Directory.Build.props```.